composer.json

```
@@ -10,7 +10,7 @@
     }
   ],
   "require": {
-    "php": "~8.1.0",
+    "php": "~8.1",
```

```suggestion
    "php": "^8.1",
```

The `^` operator is recommended (and equivalent for your change).